content/community/blm.md

```diff
@@ -14,95 +14,97 @@ summary: |
+      united to use our social, political,
+      and economic capital for the advancement of our communities.
+  - title: Colorado Freedom Fund
+    url: http://www.blackbailout.org/
     icon: icons/fist
     text: |
-      Black Visions Collective believes in a future
-      where all Black people have autonomy,
-      safety is community-led,
-      and we are in right relationship within our ecosystems.
-      BLVC is committed to a long term vision in which
-      ALL Black lives not only matter, but are able to thrive.
-
-      [Donate to Black Visions Collective »](https://secure.everyaction.com/4omQDAR0oUiUagTu0EG-Ig2)
+      is a revolving community bond fund
+      that pays ransom for our neighbors
+      unjustly detained in cages across Colorado
+      who cannot afford to buy their own freedom.
+  - title: '#BlackTransLivesMatter'
+    url: http://actforblacktranslives.org/
+    icon: icons/fist
+    text: |
+      is a collaborative effort between people
+      who stand in solidarity with Black trans women
+      and wanted to make a difference,
+      and some of the foremost Black trans leaders
+      in community and culture.
   - title: Southern Poverty Law Center
     url: https://www.splcenter.org/
     icon: icons/fist
     text: |
-      The SPLC is dedicated to fighting hate and bigotry
+      is dedicated to fighting hate and bigotry
       and to seeking justice
       for the most vulnerable members of our society.
       Using litigation, education,
       and other forms of advocacy,
       the SPLC works toward the day when the ideals of equal justice
       and equal opportunity will be a reality.
+  - title: Black Transwomen
+    url: https://blacktranswomen.org/
+    icon: icons/fist
+    text: |
+      is a national nonprofit organization
+      established to specifically address
+      the urgent concerns facing African American transwomen
+      in minority communities.
+  - title: National Bail Out
+    url: https://www.nationalbailout.org/
+    icon: icons/fist
+    text: |
+      is a Black-led and Black-centered collective
+      of abolitionist organizers, lawyers and activists
+      building a community-based movement
+      to support our folks and end systems of pretrial detention
+      and ultimately mass incarceration.
+  - title: Black Visions Collective
+    url: https://www.blackvisionsmn.org/
+    icon: icons/fist
+    text: |
+      Black Visions Collective believes in a future
+      where all Black people have autonomy,
+      safety is community-led,
+      and we are in right relationship within our ecosystems.
+      BLVC is committed to a long term vision in which
+      ALL Black lives not only matter, but are able to thrive.
 
-      [Donate to Southern Poverty Law Center »](https://donate.splcenter.org/)
-  - title: Innocence Project
-    url: https://www.innocenceproject.org/
+      [Donate to Black Visions Collective »](https://secure.everyaction.com/4omQDAR0oUiUagTu0EG-Ig2)
```

Comment:
I think we could remove this, both to be more in line with the other items listed, and because it looks like they're not currently soliciting donations.

```suggestion
```